mysql_type_plugin/__init__.py

```diff
@@ -249,6 +249,28 @@ def execute_hook(context: FunctionContext) -> Type:
                         context.api.fail(f"Could not find mysql_type.SelectResult", ct)
 
                 elif isinstance(stmt, rs.Insert):
+                    # TODO(casper): Is this how `Option<Vec<_>>` is translated: `Optional[List[_]]`?
```

Comment:
I would expect insert returning to return a List[_] not a Optional[List[_]]. But this looks the same as for select, so I guess it makes sense. Perhaps we should rename the mysql_type.SelectResult class.